src/module/PrcaOrchestrator.ts

```diff
@@ -111,4 +116,15 @@ export class PrcaOrchestrator {
         return result;
     }
 
+    /* Static methods */
+
+    public static CreatePrcaOrchestrator(logger: ILogger, collectionUrl: string, token: string, repositoryId: string, pullRequestId: number): PrcaOrchestrator {
```

Comment:
As the params are passed straight into other constructors, it is not necessary to validate them here.